ops_translator/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+#   Specifiy the cmake version range, currenty work under 3.18
+    cmake_minimum_required(VERSION 3.18)
+
+    project(OPS_TRANSLATOR)
+
+#   Copy the entire directory content to CMAKE_INSTALL_PREFIX/bin
+    INSTALL(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}" DESTINATION bin)
+
+#   The following operations are OS dependent, we do UNIX first
+    if (UNIX)
```

Comment:
any chance of doing this for windows too? We do have some users there

Reply:
I am not sure for windows, i need to check how this can be done in CMAKE